event_attribution_reporting_clicks.md

```diff
@@ -336,12 +336,13 @@ https://attributionreportto/.well-known/attribution-reporting/report-attribution
 
 The report data is included in the request body as a JSON object with the following keys:
 
+-   `attribution_destination`: the eTLD+1 that triggered attribution
```

Comment:
Could also just say the attribution_destination set on the attribution source to match the description of event id